frontend/src/services/index.ts

```diff
@@ -37,3 +37,23 @@ export async function getBookCount() {
 
   return res.data as number;
 }
+
+export type SignData = {
+  email: string;
+  password: string;
+};
+
+export async function fetchSignIn(val: SignData) {
+  const res = await axios.post(`${BASE_URL}/sign-in`, { val });
+  return res;
+}
+
+export async function fetchSignOut() {
+  const res = await axios.get(`${BASE_URL}/sign-out`);
+  return res;
+}
+
+export async function fetchHome() {
+  const res = await axios.get(`${BASE_URL}`);
+  return res;
+}
```

Comment:
Please add documentation comments for your functions. This makes it easier for others in the team to use your code and understand what it does. An example can be found [in `backend/src/models/book.ts`](https://github.com/COSC481W-2023Fall/Bookworm/blob/main/backend/src/models/book.ts#L33-L41).